.github/workflows/test.yml

```
@@ -27,6 +30,6 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install tox wheel
-    - name: Run quality checks
+    - name: Run tests and quality checks
       run: |
-        tox
+        tox p
```

```suggestion
        tox run-parallel
```

I prefer long names in scripts, because they're more descriptive.

---

Fixed (and rebased on master).